weatherman.py

```diff
@@ -0,0 +1,398 @@
+def minus_a(weather_data: WeatherData, month: int, year: int):
+    #  Getting maximum temperature column
+    col_name = weather_data.get_column_name_from_alias("max_temperature")
+    if col_name is None:
+        print("Invalid column alias")
+    else:
+        max_temp_column = weather_data.get_column_data(col_name,
+                                                       month, year)
+        print("Highest Average:", round(average(
+              max_temp_column), 2), "C")
+
+    #  Getting minimum temperature column
+    col_name = weather_data.get_column_name_from_alias("min_temperature")
+    if col_name is None:
+        print("Invalid column alias")
+    else:
+        min_temp_column = weather_data.get_column_data(col_name,
+                                                       month, year)
+        print("Lowest Average:", round(average(
+              min_temp_column), 2), "C")
+
+    #  Getting mean humidity column
+    col_name = weather_data.get_column_name_from_alias("mean_humidity")
+    if col_name is None:
+        print("Invalid column alias")
+    else:
+        mean_humidity_column = weather_data.get_column_data(col_name,
+                                                            month, year)
+        print("Average Mean Humidity:",
+              round(average(mean_humidity_column), 2), "%")
+    return
+
+
+# Shows a graph of highest and lowest weather of a month
+def minus_c(weather_data: WeatherData, month: int, year: int):
+    #  Getting maximum temperature column
+    col_name = weather_data.get_column_name_from_alias("max_temperature")
+    if col_name is None:
+        print("Invalid column alias")
+        sys.exit()
+    else:
+        max_temp_column = weather_data.get_column_data(col_name,
+                                                       month, year)
+
+    #  Getting minimum temperature column
+    col_name = weather_data.get_column_name_from_alias("min_temperature")
+    if col_name is None:
+        print("Invalid column alias")
+        sys.exit()
+    else:
+        min_temp_column = weather_data.get_column_data(col_name,
+                                                       month, year)
+    max_len = len(max_temp_column)
+    min_len = len(min_temp_column)
+    len_ = max_len
+    if max_len != min_len:
+        if min_len < max_len:
+            len_ = min_len
+        print_str = "\033[1;31;40m Note: Unequal data of maximum"
+        print_str += " and minimum temperature"
+        print(print_str)
+    for i in range(0, len_):
+        suffix_text = str("\033[0;37;40m%.2d " % (i+1))
+        low_print = "\033[0;34;40m" + "+"*abs(int(min_temp_column[i]))
+        high_print = "\033[0;31;40m" + "+"*abs(int(max_temp_column[i]))
+        add_text = "\033[0;34;40m "
+        add_text += (str(min_temp_column[i]) + "C\033[1;37;40m-")
+        add_text += ("\033[0;31;40m" + str(max_temp_column[i]) + "C")
+        text = suffix_text + low_print + high_print + add_text
+        print(text)
```

Comment:
better variable names needed here, maybe move this to a seperate method as well.